### PR TITLE
(DOCSP-11394): Update Punctuation page

### PR DESCRIPTION
**Pull request by melissamahoney-mongodb**
[JIRA](https://jira.mongodb.org/browse/DOCSP-11394)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11394/style-guide/style/punctuation.html)

### CODE REVIEW

File: source/style-guide/style/punctuation.txt [hunk 1]
```diff
@@ -4,12 +4,10 @@
 Punctuation
 ===========
 
-.. include:: /style-guide/includes/page-needs-update.rst
-
 Use punctuation correctly and consistently. This section provides
 guidelines for using punctuation in MongoDB documentation. For basic
 rules about punctuation, see a grammar reference, such as the
-*Harbrace College Handbook*.
+*Chicago Manual of Style*.
```
Comment (atsansone) [inline source/style-guide/style/punctuation.txt:10]: **suggestion:** CMOS should be our official reference. 
```
see :cmos17:`section 5, Grammar and Usage </part2/ch05/toc.html>` in the *Chicago Manual of Style*.

File: source/style-guide/style/punctuation.txt [hunk 2]
```diff
@@ -63,7 +61,7 @@ Commas
 ~~~~~~
 
 Use the following guidelines for commas. For basic comma usage, see a
-grammar reference, such as the *Harbrace College Handbook*.
+grammar reference, such as the *Chicago Manual of Style*.
```
Comment (atsansone) [inline source/style-guide/style/punctuation.txt:64]: **suggestion:**
```
:cmos17:`section 6.16, Commas </part2/ch06/psec016.html>` in the Chicago Manual of Style.
```

File: source/style-guide/style/punctuation.txt [hunk 3]
```diff
@@ -116,10 +114,10 @@ grammar reference, such as the *Harbrace College Handbook*.
        However, you can easily update the version by using the
        WordPress management dashboard.
 
-       Unlike the other alarms in this list, you set the network check alarm
-       variable upon network check creation.
+       Unlike the other alarms in this list, you set the network check
+       alarm variable upon network check creation.
 
-       For more information, see Upgrading your Private Cloud.
+       For more information, see Upgrading Ops Manager.
```
Comment (atsansone) [inline source/style-guide/style/punctuation.txt:120]: **suggestion:**

```suggestion
       To learn more, see Upgrading Ops Manager.
```

File: source/style-guide/style/punctuation.txt [hunk 4]
```diff
@@ -130,7 +128,7 @@ grammar reference, such as the *Harbrace College Handbook*.
        Unlike the other alarms in this list you set the network check
        alarm variable upon network check creation.
 
-       For more information see Upgrading your Private Cloud.
+       For more information see Upgrading Ops Manager.
```
Comment (atsansone) [inline source/style-guide/style/punctuation.txt:131]: **suggestion:**

```suggestion
       To learn more, see Upgrading Ops Manager.
```

File: source/style-guide/style/punctuation.txt [hunk 5]
```diff
@@ -347,7 +344,7 @@ Periods
 ~~~~~~~
 
 Use the following guidelines for periods. For basic period usage, see a
-grammar reference, such as the *Harbrace College Handbook*.
+grammar reference, such as the *Chicago Manual of Style*.
```
Comment (atsansone) [inline source/style-guide/style/punctuation.txt:347]:
```suggestion
grammar reference, :cmos17:`section 6.12, Periods </part2/ch06/psec012.html>` in the *Chicago Manual of Style*.
```

File: source/style-guide/style/punctuation.txt (hunk 1 above)
Comment (melissamahoney-mongodb) [reply source/style-guide/style/punctuation.txt:10]: I agree, just missed it on this page. Thanks for 👀

**Comment by melissamahoney-mongodb** [comment]
> Would recommend the CMOS links, but otherwise LGTM.

I did add the CMOS links.

**Pull request merged**